src/conditions-n-loops-tasks.js

```diff
@@ -10,69 +10,66 @@
 
 /**
- * Returns a boolean value whether a number is positive. Relate zero to positive numbers.
- * In this task, the use of methods of the Number and Math classes is not allowed.
+ * Determines whether a given number is positive. Zero is considered positive.
+ * This function does not use Number or Math class methods.
  *
- * @param {number} number
- * @return {boolean}
+ * @param {number} number - The number to check.
+ * @return {boolean} True if the number is positive or zero, false otherwise.
  *
  * @example:
  *  10 => true
  *  0  => true
  *  -5 => false
  */
 function isPositive(/* number */) {
-  throw new Error('Not implemented');
+  throw new Error('Function isPositive is not yet implemented.');
 }
 
 /**
- * Returns the maximum of three numbers.
- * In this task, the use of methods of the Array class is not allowed.
+ * Returns the maximum of three numbers without using Array class methods.
  *
- * @param {number} a
- * @param {number} b
- * @param {number} c
- * @return {number}
+ * @param {number} a - The first number.
+ * @param {number} b - The second number.
+ * @param {number} c - The third number.
+ * @return {number} The maximum of the three numbers.
  *
  * @example:
  *  1, 2, 3       => 3
  *  -5, 0, 5      => 5
  *  -0.1, 0, 0.2  => 0.2
  */
 function getMaxNumber(/* a, b, c */) {
-  throw new Error('Not implemented');
+  throw new Error('Function getMaxNumber is not yet implemented.');
 }
 
 /**
- * Returns boolean value true if the queen can hit the king next move. If it cannot, false is returned.
- * The size of the field is 8 * 8 cells.
- * See more details at https://en.wikipedia.org/wiki/Queen_(chess)
- *
+ * Checks if a queen can capture a king in the next move on an 8x8 chessboard.
+ *  * See more details at https://en.wikipedia.org/wiki/Queen_(chess)
  * @typedef {{
  *  x: number,
  *  y: number
  * }} Position
- * @param {Position} queen
- * @param {Position} king
- * @return {boolean}
+ * @param {Object} queen - The position of the queen.
+ * @param {Object} king - The position of the king.
+ * @return {boolean} True if the queen can capture the king, false otherwise.
  *
- * @example:
+ * @example
  * {x: 1, y: 1}, {x: 5, y: 5} => true
  * {x: 2, y: 1}, {x: 2, y: 8} => true
  * {x: 1, y: 1}, {x: 2, y: 8} => false
  * {x: 1, y: 1}, {x: 2, y: 8} => false
  */
-function isCheckmate(/* queen, king */) {
-  throw new Error('Not implemented');
+function canQueenCaptureKing(/* queen, king */) {
+  throw new Error('Function canQueenCaptureKing is not yet implemented.');
 }
 
 /**
- * Returns boolean whether the triangle is isosceles.
+ * Determines whether a triangle is isosceles based on its side lengths.
  *
- * @param {number} a
- * @param {number} b
- * @param {number} c
- * @return {boolean}
+ * @param {number} a - Length of the first side.
+ * @param {number} b - Length of the second side.
+ * @param {number} c - Length of the third side.
```

Comment:
You can add articles here as well to make it more consistent with the rest of the descriptions.
**The** length of